src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Validators/AddDraftApprenticeshipViewModelValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using SFA.DAS.ProviderCommitments.Web.Models;
+
+namespace SFA.DAS.ProviderCommitments.Web.Validators
+{
+    public class AddDraftApprenticeshipViewModelValidator : AbstractValidator<AddDraftApprenticeshipViewModel>
+    {
+        public AddDraftApprenticeshipViewModelValidator()
+        {
+            //Example validation rules for testing
+            //RuleFor(x => x.FirstName).NotEmpty();
+            //RuleFor(x => x.LastName).NotEmpty();
+            //RuleFor(x => x.BirthDate).Must(y => y.IsValid).WithMessage("That date is not valid").When(z => z.BirthDate.HasValue);
+            //RuleFor(x => x.StartDate).Must(y => y.IsValid).WithMessage("That date is not valid").When(z => z.StartDate.HasValue);
+            //RuleFor(x => x.FinishDate).Must(y => y.IsValid).WithMessage("That date is not valid").When(z => z.FinishDate.HasValue);
+        }
+    }
+}
```

Comment:
With us deciding to do validation elsewhere, it probably makes sense to remove this file to avoid confusion in the future.